code/datums/security_state.dm

```diff
@@ -121,6 +121,8 @@
 		previous_security_level.switching_down_from()
 		new_security_level.switching_down_to()
 
+	for(var/obj/item/weapon/rig/R as anything in GLOB.rig_update_tracking)
```

Comment:
what is the gameplay purpose of this auto update thing ? It seems like it makes life more difficult for antags

Reply:
As you mentioned in discord RIG tracking level should be randomised. If you won't turn on automatic mode it won't do anything.